carcara/src/parser/error.rs

```diff
@@ -100,6 +123,19 @@ where
     }
 }
 
+/// Returns an error if the value of of `sequence` is not in the `expected` range.
+pub fn assert_args_value<T, R>(sequence: &[T], range: R) -> Result<(), ParserError>
+where
+    R: Into<Range>,
+{
+    let range = range.into();
+    if range.contains(sequence.len()) {
+        Ok(())
+    } else {
+        Err(ParserError::WrongNumberOfArgs(range, sequence.len()))
+    }
+}
+
```

Comment:
function mentioned to check args value. The implementation is not there yet, I just copied the body from `assert_num_args`